database/languages/cplusplus.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "name": "c++ official",
+        "description": "It is an official website of cplusplus where you can get tutorial that provides detailed explanation, examples, and tasks of c++ language.",
+        "url": "https://cplusplus.com/doc/tutorial/",
+        "category": "Other",
+        "subcategory": "cplusplus"
+    },
+    {
+        "name": "c++ tutorialspoint ",
+        "description": "It is an website of cplusplus where you can get tutorial that provides detailed explanation, examples, and tasks of c++ language.",
+        "url": "https://www.tutorialspoint.com/cplusplus/index.htm",
+        "category": "Other",
+        "subcategory": "cplusplus"
```

Comment:
Hmmm... 🤔, is this a tutorial on how to use C++ or a project where knowledge of this language is required?  If it's the former, I highly recommend replacing this with a source that describes my second idea.